pest-ion/src/bin/pest2ion/main.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates.
+
+use anyhow::Result;
+use clap::{crate_authors, crate_description, crate_version, App, Arg, ArgGroup};
+use ion_rs::value::writer::{ElementWriter, Format, TextKind};
+use pest_ion::{from_read, TryPestToElement};
+use std::fs::File;
+use std::io::{stdin, stdout, Write};
+use std::path::Path;
+
+fn main() -> Result<()> {
+    let matches = App::new("Pest to Ion Converter")
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about(crate_description!())
+        .arg(
+            Arg::with_name("FILE")
+                .help("The input file to parse (defaults to STDIN)")
+                .index(1),
+        )
+        .arg(
+            Arg::with_name("OUTPUT_FILE")
+                .long("output")
+                .short("o")
+                .takes_value(true)
+                .help("Writes output to the given file (defaults to STDOUT)"),
+        )
+        .arg(
+            Arg::with_name("text")
+                .long("text")
+                .short("t")
+                .help("Generate Ion text (default)"),
+        )
+        .arg(
+            Arg::with_name("binary")
+                .long("binary")
+                .short("b")
+                .help("Generate Ion binary"),
+        )
```

Comment:
Not sure what the use-case for outputting binary ion from this tool is...  Do you expect the size benefits of the binary format be of significant benefit?

Reply:
The use case is actually giving me a way to work around #43 (which apparently there is a bug with serializing deeply nested Ion).  #37 probably helps, but I see no reason not to give a little flexibility here.